src/main/scala/io/scalac/tezos/translator/TranslationsService.scala

```diff
@@ -0,0 +1,29 @@
+package io.scalac.tezos.translator
+
+import io.scalac.tezos.translator.model.{History, Translation}
+
+import scala.collection.mutable
+
+class TranslationsService {
+
+  private val maxReturnSize = 10
+  private val maxSize = 100
+
+  private val translations = mutable.MutableList[History]()
```

Comment:
is it thread safe?

Reply:
No, it's not thread-safe. But it's a temporary solution, just to make API workable for frontend devs. Probably (there is no decision yet) it'll be stored in DB, if not e.g. I'll hide this list inside an actor.